ui/__tools__/dev-server.js

```diff
@@ -11,13 +11,19 @@ const webpackConfig = require('../__config__/webpack.config.dev')({
 const compiler = webpack(webpackConfig);
 const app = express();
 
+const fontDefinitions = require('../__helpers__/fontDefinitions');
```

Comment:
Can we move this to the top?

Reply:
maybe a comment to say it's temporary too?